src/Akka.Monitoring.ApplicationInsights.Demo/Program.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Threading;
+using Akka.Actor;
+
+namespace Akka.Monitoring.ApplicationInsights.Demo
+{
+    class HelloActor : TypedActor, IHandle<string>
+    {
+        protected override void PreStart()
+        {
+            Context.IncrementActorCreated();
+            base.PreStart();
+        }
+
+        protected override void PostStop()
+        {
+            Context.IncrementActorStopped();
+            base.PostStop();
+        }
+
+        public void Handle(string message)
+        {
+            Context.Timing("helloActorTiming", 1030);
+            Context.IncrementMessagesReceived();
+            Console.WriteLine("Received: {0}", message);
+            if (message == "Goodbye")
+            {
+                Context.Self.Tell(PoisonPill.Instance);
+                Program.ManualResetEvent.Set(); //allow the program to exit
+            }
+            else
+                Sender.Tell("Hello!");
+        }
+    }
+
+    class GoodbyeActor : TypedActor, IHandle<Tuple<IActorRef, string>>, IHandle<string>
+    {
+        protected override void PreStart()
+        {
+            Context.IncrementActorCreated();
+            base.PreStart();
+        }
+
+        protected override void PostStop()
+        {
+            Context.IncrementActorStopped();
+            base.PostStop();
+        }
+
+        public void Handle(string message)
+        {
+            Context.IncrementMessagesReceived();
+            Console.WriteLine("Received: {0}", message);
+            Sender.Tell("Goodbye");
+            Context.Self.Tell(PoisonPill.Instance);
+        }
+
+        public void Handle(Tuple<IActorRef, string> message)
+        {
+            Context.IncrementMessagesReceived();
+            message.Item1.Tell("Starting");
+        }
+    }
+
+    class Program
+    {
+        public static AutoResetEvent ManualResetEvent = new AutoResetEvent(false);
+
+        private static ActorSystem _system;
+
+        static void Main(string[] args)
+        {
+
+            _system = ActorSystem.Create("akka-performance-demo");
+            var registeredMonitor = ActorMonitoringExtension.RegisterMonitor(
+                _system, new ActorAppInsightsMonitor(
+                    "<INSERT YOUR KEY>", true));
```

Comment:
Oh cool, so will this just work locally then so long as you have a key? Wasn't sure if the app had to literally be running in Service Fabric environment in order to work.